local_postgres_docker_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Couldn't you just run a [postgres container](https://hub.docker.com/_/postgres/) instead of dealing with all the networking to localhost?